sherlock/resources/data.json

```diff
@@ -2213,6 +2213,55 @@
     "url": "http://uid.me/{}",
     "urlMain": "https://uid.me/",
     "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
+    "username_unclaimed": "noonewouldeveethis7"
+  },
+  "TensorFlow": {
+    "errorType": "status_code",
+    "url": "https://www.tensorflow.org/",
```

Comment:
Your urls miss `{}` parts used for username substitution. Take a look at https://github.com/sherlock-project/sherlock/wiki/Adding-Sites-To-Sherlock.

You can also run tests for your sites locally with:
```bash
> python3 -m unittest tests.all --verbose
```